ConsoleAppfForGit/Pizza/Program.cs

```diff
@@ -26,7 +26,7 @@ static void Main(string[] args)
                 Console.WriteLine("Added ingredients: {0}", pizza.Ingredients[i]);
             }
 
-            //Comment
+           
 
             Console.ReadLine();
         }
```

Comment:
As a Pizza consumer I'd be interested in getting the list of components as a nice csv string so that I can use it as I want somewhere else. So please enhance Pizza so that it may return its components as a csv string.